src/System.Windows.Forms/src/System/Windows/Forms/Formatter.cs

```diff
@@ -429,64 +432,65 @@ private static CultureInfo GetFormatterCulture(IFormatProvider formatInfo)
         {
             try
             {
-                MethodInfo mi;
-
-                mi = targetType.GetMethod("Parse",
-                                        BindingFlags.Public | BindingFlags.Static,
-                                        null,
-                                        new Type[] { stringType, typeof(NumberStyles), typeof(IFormatProvider) },
-                                        null);
+                MethodInfo? mi = targetType.GetMethod(
+                    "Parse",
+                    BindingFlags.Public | BindingFlags.Static,
+                    null,
+                    new Type[] { stringType, typeof(NumberStyles), typeof(IFormatProvider) },
+                    null);
                 if (mi is not null)
                 {
-                    return mi.Invoke(null, new object[] { (string)value, NumberStyles.Any, formatInfo });
+                    return mi.Invoke(null, new object?[] { (string?)value, NumberStyles.Any, formatInfo });
                 }
 
-                mi = targetType.GetMethod("Parse",
-                                        BindingFlags.Public | BindingFlags.Static,
-                                        null,
-                                        new Type[] { stringType, typeof(IFormatProvider) },
-                                        null);
+                mi = targetType.GetMethod(
+                    "Parse",
+                    BindingFlags.Public | BindingFlags.Static,
+                    null,
+                    new Type[] { stringType, typeof(IFormatProvider) },
+                    null);
                 if (mi is not null)
                 {
-                    return mi.Invoke(null, new object[] { (string)value, formatInfo });
+                    return mi.Invoke(null, new object?[] { (string?)value, formatInfo });
                 }
 
-                mi = targetType.GetMethod("Parse",
-                                        BindingFlags.Public | BindingFlags.Static,
-                                        null,
-                                        new Type[] { stringType },
-                                        null);
+                mi = targetType.GetMethod(
+                    "Parse",
+                    BindingFlags.Public | BindingFlags.Static,
+                    null,
+                    new Type[] { stringType },
+                    null);
                 if (mi is not null)
                 {
-                    return mi.Invoke(null, new object[] { (string)value });
+                    return mi.Invoke(null, new object?[] { (string?)value });
                 }
 
                 return parseMethodNotFound;
             }
             catch (TargetInvocationException ex)
             {
-                throw new FormatException(ex.InnerException.Message, ex.InnerException);
+                throw new FormatException(ex.InnerException!.Message, ex.InnerException);
```

Comment:
I do not see any reason why we can not make this change here. 

```suggestion
                throw new FormatException(ex.InnerException?.Message, ex.InnerException);
```